src/components/Utils/__snapshots__/EntityURLControl.spec.ts.snap

```diff
@@ -50,7 +50,7 @@ exports[`EntityURLControl.vue renders snapshot 1`] = `
         </span>
         
         
-         Copy URL
+         Copy URL 
```

Comment:
This is an expected consequence of how snapshots are generated in combination with how HTML renders white space. Consider the following code:

```html
<span>
  text node</span>
```

The HTML parser will render this as " text node" (side note: GitHub annoyingly `.trim()`s white space from code blocks so I can’t code-format this). This leads to snapshots reproducing this kind of whitespace because it accurately represents browsers would render.